src/cloud-resources/src/crd/materialize.rs

```diff
@@ -194,6 +194,17 @@ pub mod v1alpha1 {
         #[serde(default)]
         pub environment_id: Uuid,
 
+        // The name of a configmap containing system_parameters in json format
+        // NOTE: The configmap must contain a `materialize.json` key whose value
+        // is a valid json containing valid system parameters.
+        // ex:
+        // data:
+        //   materialize.json: |
+        //     {
+        //       "max_connections": 10
+        //     }
+        pub system_parameter_configmap_name: Option<String>,
```

Comment:
See above, this comment doesn't match what you are doing above in the mzcompose materialized.py service file.